executor/show_test.go

```diff
@@ -170,18 +170,66 @@ func (s *testSuite2) TestShow2(c *C) {
 	// TODO: In MySQL, the result is "autocommit ON".
 	tk2.MustQuery("show global variables where variable_name = 'autocommit'").Check(testkit.Rows("autocommit 1"))
 
+	// TODO: specifying charset for national char/varchar should not be supported
```

Comment:
s/specifying/Specifying
Please add `.` at the end of a line.

Reply:
done